### DISCUSSION TITLE
posts/%E8%AE%BA%E6%83%B0%E6%80%A7

### DISCUSSION THREAD

**Discussion by giscus** [martin-liu/martin-liu.github.io#14] [Announcements]
# posts/%E8%AE%BA%E6%83%B0%E6%80%A7

惰性是可怖的，它抗拒改变，因而也抗拒学习、精益和成长，它会侵蚀智慧，令珠玉斑驳，令理想沉沙。而或许是普遍的不幸，在我们几经流转的人生，惰与勤辗转往复，见证了所有的过往和悲欢。

http://localhost:3000/posts/%E8%AE%BA%E6%83%B0%E6%80%A7

**Comment by martin-liu** [comment]
test
